Samples/MedicationsTracker/MedicationsTracker/Transitioner.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Transitioner.swift
+//  MedicationsTracker
+//
+// Copyright (c) 2017 Microsoft Corporation. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+
+struct Transitioner
+{
+    
+    /*
+     * Handles the transition logic for the stored view controller
+     */
+    func transition()
+    {
+        
+    }
```

Comment:
I think you'll need some changes when you get to these, at least passing in the navigationController for pushing the next page view controller.  (But no need to do  that until you get to creating the views and navigating between)